### ISSUE TITLE
Would like to help with PDF-Export

### ISSUE THREAD

**Issue by m-art-in** [Daniel-KM/Omeka-S-module-Ebook#8]
Thank you for this great module. I have seen that the generation of PDFs has not yet been implemented. I would like to support this as a developer. I have seen in the code that the form is already prepared to optionally select PDF as output format. 

I'm not sure if you already have a PDF library in mind. There is probably no corresponding package in the vendor folder yet. ImageMagick would work theoretically, but is very slow. There is the Python library img2pdf, which is very fast, but only creates PDFs from images. What would be a possibility for PHP. I would be happy if I could get some tips and at the same time advance the module.

My goal would be to integrate the images into the PDF. The epub only links to them.

**Comment by m-art-in** [comment]
Update:
I will try it with [TCPDF](https://github.com/tecnickcom/tcpdf) and added to my fork in composer,json

`require : { .... 'tecnickcom/tcpdf': '^6.3' }`

I don't know this library yet and can't estimate if it will work, but I will try it. 
Within the Zend Framework there is also the possibility to create PDF. Maybe this is more stable?

**Comment by Daniel-KM** [comment]
Yes, it's a good idea. Currently, the module creates ebook only, even if there is an option for pdf. This options was used to make the relation with the common pdftk command line tool, and there is a pdf library mikehaertl/php-pdftk for that, and I used it in some other modules too ([BulkImport](Daniel-KM/Omeka-S-module-BulkImport)). The main advantage of your option is that it doesn't depends of a command line tool, so it's more simple to use. I don't know the performance of it.
